src/libaktualizr/package_manager/dockerappmanager.cc

```diff
@@ -145,6 +145,11 @@ data::InstallationResult DockerAppManager::install(const Uptane::Target &target)
   data::InstallationResult res;
   Uptane::Target current = OstreeManager::getCurrent();
   if (current.sha256Hash() != target.sha256Hash()) {
+    // notify the bootloader before installation happens as it is not atomic
+    // and a false notification doesn't hurt with rollback support in place
+    if (bootloader_ != nullptr) {
+      bootloader_->updateNotify();
+    }
```

Comment:
we should title this commit `[fio toup]` so we remember to push it upstream.

Reply:
Sure, will change.